third_party/cpgfunctionEP-0.3.2/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+cmake_minimum_required(VERSION 3.6)
```

Comment:
The CMake lists reference local libraries only. There is nothing additional a user needs to have installed to make use of UBHWTcalc. If the project is built by E+ the internal tests will not be built. The internal test files and library (nlohmann json) are not included in this copy of the library.